Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# DNA for Greymane, Fordring, VanCleef, Wrynn + Others

## Changelog:
- Made very small fixes to **Anasterian**, **Lor'themar**, and **Alleria**. _(These are barely noticeable so they are not included in the list.)_
- Added **15** new character portraits:

_[Link to DNA spreadsheet](https://docs.google.com/spreadsheets/d/1Cf0FGh_iNoyDsIwceDkL_Fr4B7_rOZKaa0QbzLNqmYQ/edit#gid=652876950) for additional information, such as what was used as reference for the portraits._

### Gnomes

 **Mekkatorque**
- Gelbin Mekkatorque (by @ercarp)

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/78575425/130371735-91febea9-14e6-46f6-9e1a-13ec3ed1f838.png)

</details>

### Humans

**Greymane**
- Tess Greymane (by @ercarp)
- Liam Greymane (by @ercarp)
- Mia Greymane (by @ercarp)
- Archibald Greymane (by @ercarp)

<details><summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/78575425/130371308-f9e94cfe-0bd0-4637-9304-e68e3ea9dbf2.png)
![image](https://user-images.githubusercontent.com/78575425/130371315-74e8f4c8-826b-426f-83c5-2e90ce7b7de6.png)
![image](https://user-images.githubusercontent.com/78575425/130371318-62f7890a-d3c6-4f44-941f-45ee54537344.png)
![image](https://user-images.githubusercontent.com/78575425/130371322-d72b9850-e1aa-498b-ad48-1d9f8ccac2a2.png)

</details>

**Fordring**
- _**Reworked**_ Tirion Fordring (by @ercarp)
- Taelan Fordring (by @ercarp)
- Karandra Fordring (by @ercarp)

<details><summary>Screenshots</summary>

**New Tirion DNA**
![image](https://user-images.githubusercontent.com/78575425/130371340-17240f8e-8280-48bd-9ee6-4292fba4eef0.png)

**Old Tirion DNA**
_He looked like he was angry all the time so I adjusted his brows a bit._
![image](https://user-images.githubusercontent.com/78575425/130372740-89e9afa9-8839-4a3a-8ae5-7291d1716906.png)

![image](https://user-images.githubusercontent.com/78575425/130371348-5e1145d9-61d1-4d38-839f-7932ff16fd1d.png)
![image](https://user-images.githubusercontent.com/78575425/130371354-3031b159-af15-47da-af1a-267cd047cfc8.png)

</details>

**Menethil**
- _**Reworked**_ Arthas Menethil (by @ercarp)

<details><summary>Screenshot</summary>

**New DNA**
![image](https://user-images.githubusercontent.com/78575425/130371377-52c2c943-9d65-4dd6-8675-ca9c2fc923ed.png)

**Old DNA**
![image](https://user-images.githubusercontent.com/78575425/130372706-5c6540b7-c4b0-44b8-b0b8-4c0497d16f2c.png)

</details>

**VanCleef**
- Edwin VanCleef (by @ercarp)
- Vanessa VanCleef (by @ercarp)

<details><summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/78575425/130371391-2bbf2cec-90bd-45d2-9d71-d6a4229f3872.png)
![image](https://user-images.githubusercontent.com/78575425/130371406-030dcdba-1bfd-4baa-8b4b-05550a2b5340.png)

</details>

**Wrynn**
- Landan Wrynn (by @ercarp)
- Barathen Wrynn (by @ercarp)
- Varia Wrynn (by @ercarp)
- Taria Wrynn (by @ercarp)

<details><summary>Screenshots</summary>

![image](https://user-images.githubusercontent.com/78575425/130371417-ea7d7ef1-e90a-4297-af89-f049ff58abda.png)
![image](https://user-images.githubusercontent.com/78575425/130371423-57541bc8-3bfb-4f61-92c5-b61450f0bb6f.png)
![image](https://user-images.githubusercontent.com/78575425/130371433-09d7d953-787e-4455-a02d-e2fc429301e0.png)
![image](https://user-images.githubusercontent.com/78575425/130371439-19eb8b8a-0ebe-4d0d-b2ba-14bcea43b7a3.png)

</details>

# Tasks:
- [x] `is_taelan_trigger` used in `is_any_historical_character_trigger`


# How to test:
Consider the following when giving your feedback:
- **Canonicity**: Observe if the character's appearance matches the information provided by canon in terms of hair, eye, and skin color; hairstyle, facial hair; body shape, etc.
- **Familiality**: Observe whether or not characters look sufficiently related to each other if they have family members with custom DNA.
- **General Feedback**: Give general feedback on how you feel about each addition. In simple terms: does the portrait look good to you personally?

**How to view the portraits**:

- You can either **A)** rely on the screenshots provided above (although most portraits look much better once you see them in motion) or **B)** have a look at the portraits in-game.
  - Enable [Debug Mode](https://ck3.paradoxwikis.com/Console_commands#Enabling_debug_mode) and use the `age [amount] [character id]` console command to change a character's age if they are too young or old in the current bookmark.
